src/test/java/de/retest/recheck/report/TestReportFilterTest.java

```diff
@@ -0,0 +1,208 @@
+package de.retest.recheck.report;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import de.retest.recheck.ignore.Filter;
+import de.retest.recheck.review.ignore.AttributeFilter;
+import de.retest.recheck.suite.ExecutableSuite;
+import de.retest.recheck.ui.descriptors.Element;
+import de.retest.recheck.ui.descriptors.GroundState;
+import de.retest.recheck.ui.descriptors.IdentifyingAttributes;
+import de.retest.recheck.ui.descriptors.RootElement;
+import de.retest.recheck.ui.diff.AttributeDifference;
+import de.retest.recheck.ui.diff.AttributesDifference;
+import de.retest.recheck.ui.diff.DurationDifference;
+import de.retest.recheck.ui.diff.ElementDifference;
+import de.retest.recheck.ui.diff.IdentifyingAttributesDifference;
+import de.retest.recheck.ui.diff.RootElementDifference;
+import de.retest.recheck.ui.diff.StateDifference;
+import de.retest.recheck.ui.image.Screenshot;
+
+class TestReportFilterTest {
+	Filter filter;
```

Comment:
Are all these fields really necessary? Or can some be transformed to local variables?